bees.schema.json

```
@@ -20,18 +20,21 @@
             "type": "string"
           },
           "flowers_accepted": {
-            "type": "string"
+            "type": "array"
```

The `flowers_accepted` array should only contain strings and they should be unique.
```suggestion
            "type": "array",
            "uniqueItems": true,
            "items": {
            	"type": "string"
            }
```